src/app/(profile)/(components)/ProfileCard/ProfileCard.tsx

```diff
@@ -85,9 +87,20 @@ const ProfileCard = ({
             <Avatar src={data.avatarUrl || ''} size={78} />
           </div>
           <div className={styles.data}>
-            <p className={styles.name}>
-              {highlightText(data.fullName, searchTerm)}
-            </p>
+            <div className={styles.nameContainer}>
+              <p className={styles.name}>
+                {highlightText(data.fullName, searchTerm)}
+              </p>
+              {isHiddenName ? (
+                <div className={styles.nameCover}>
+                  <div className={styles.paddlockContainer}>
+                    <div className={styles.paddlockTop}></div>
+                    <div className={styles.paddlockBottom}></div>
+                  </div>
+                  Login to access
+                </div>
+              ) : null}
+            </div>
```

Comment:
We need to make it as usual. I mean you have to copy the svg from figma and create icon component eg. `PaddlockIcon.tsx` inside `src/assets/icons`. Then, use it as normal component against the paddlock divs. Also, you can return `higlightText` when `isHiddenName` is false againts null.

Reply:
Agree!